go/felix/rules/nat.go

```diff
@@ -36,3 +39,47 @@ func (r *DefaultRuleRenderer) NATOutgoingChain(natOutgoingActive bool, ipVersion
 		Rules: rules,
 	}
 }
+
+func (r *DefaultRuleRenderer) DNATsToIptablesChains(dnats map[string]string) []*iptables.Chain {
+	// Extract and sort map keys so we can program rules in a determined order.
+	sortedExtIps := make([]string, 0, len(dnats))
+	for extIp, _ := range dnats {
```

Comment:
`, _` isn't needed here, you only need the `_`  at the start if you want the second value of the "tuple"